runtime/services/autocli.go

```diff
@@ -35,4 +62,25 @@ func (a AutoCLIQueryService) AppOptions(context.Context, *autocliv1.AppOptionsRe
 	}, nil
 }
 
+type autocliConfigurator struct {
```

Comment:
Nit move that down next to its methods.
Can you explain the use case?

Reply:
This is just so that we can call `RegisterServices` to discover the module's service names.

Reply:
updated

Reply:
I guess this PR using the configurator approach is an intermediary solution, right? I'm in favor of removing the `Configurator` interface ultimately (since we're going the interfaces approach)

AppModule's RegisterServices should be simply `RegisterServices(grpc.ServiceRegistrar)`, which would remove `autocliConfigurator` from this file.

Reply:
Exactly